addons/default/visiosoft/advs-module/resources/views/ad-detail/partials/content.twig

```diff
@@ -37,7 +37,9 @@
                 </div>
 
                 <!-- Content With Block -->
-                {{ addBlock('ad-detail/content',{'features':features,'adv':adv, 'profile': profile})|raw }}
+                <div id="nav-comments" class="tab-pane fade in">
+                    {{ addBlock('ad-detail/content',{'features':features,'entry':adv, 'profile': profile, 'commenturl': 'advs::saveComment', 'entry_type': 'Visiosoft\\AdvsModule\\Adv\\AdvModel'})|raw }}
+                </div>
```

Comment:
You can move the comment url to the comment module. why are you sending the entry_type!?